code/game/turfs/exterior/exterior_grass.dm

```diff
@@ -0,0 +1,35 @@
+/turf/exterior/grass
+	name = "grass"
+	icon = 'icons/turf/exterior/grass.dmi'
+	icon_edge_layer = 0.03
+	color = "#799c4b"
+	footstep_type = /decl/footsteps/grass
+
+/turf/exterior/grass/Initialize()
+	. = ..()
+	if(GLOB.using_map.use_overmap)
+		var/obj/effect/overmap/visitable/sector/exoplanet/E = map_sectors["[z]"]
+		if(istype(E) && E.grass_color)
+			color = E.grass_color
+	/*
```

Comment:
why keep this commented out

Reply:
Because exterior turf doesn't have the vars anymore. I should label this as having deps, it relies on the extension mining PR.